wire-ios/Wire-iOS/Sources/UserInterface/Conversation/Content/ConversationTableViewDataSource.swift

```diff
@@ -181,6 +182,35 @@ final class ConversationTableViewDataSource: NSObject {
         super.init()
 
         tableView.dataSource = self
+
+        setupObservers()
+    }
+
+    private func setupObservers() {
+
+        NotificationCenter.default.addObserver(
+            self,
+            selector: #selector(willEnterForeground),
+            name: UIApplication.willEnterForegroundNotification,
+            object: nil
+        )
+
+        NotificationCenter.default.addObserver(
+            self,
+            selector: #selector(didEnterBackground),
+            name: UIApplication.didEnterBackgroundNotification,
+            object: nil
+        )
+    }
+
+    @objc
+    private func willEnterForeground(_ notification: NSNotification) {
+        fetchController?.delegate = self
+    }
+
+    @objc
+    private func didEnterBackground(_ notification: NSNotification) {
+        fetchController?.delegate = nil
```

Comment:
suggestion: add some comments explaining why we do this.

Reply:
done